todo/webpack.config.js

```diff
@@ -0,0 +1,26 @@
+const path = require('path');
+
+const HtmlWebpackPlugin = require('html-webpack-plugin');
```

Comment:
`html-webpack-plugin`  injects js to specified html file.  Need to configure 
`template` : Skeleton of the html template
`filename` : Generate new html file by injecting specified js